Content.Client/Lathe/UI/LatheMaterialsEjectionMenu.xaml.cs

```diff
@@ -0,0 +1,74 @@
+using Content.Shared.Materials;
+using Robust.Client.AutoGenerated;
+using Robust.Client.GameObjects;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Prototypes;
+using System.Linq;
+
+namespace Content.Client.Lathe.UI
```

Comment:
Current conventions prefer file-scoped namespaces: https://docs.spacestation14.io/en/getting-started/conventions#file-layout. Please switch to those.

Reply:
Good catch, I missed that